operator-framework-core/src/main/java/io/javaoperatorsdk/operator/Operator.java

```diff
@@ -62,10 +65,14 @@ public ConfigurationService getConfigurationService() {
    */
   @SuppressWarnings("unchecked")
   public void start() {
-    synchronized (lock) {
+    try {
+      lock.lock();
```

Comment:
Why is this method synchronized at all? Do we except this to be called from a multi-threaded environment? I't not enough just to say in docs that this is not thread safe, call it from single thread?

Reply:
It's actually something I've wondered about. It's from a different PR. I don't think that it's called from multiple threads since it's just called when the app starts.